doc/releases/bobtail.rst

```diff
@@ -97,7 +97,7 @@ Notable changes
 * rgw: fix multipat upload listing
 * rgw: don't copy object when copied onto self
 * osd: fix caps parsing for pools with - or _
-* osd: allow pg log trimming when degraded, scrubbing, recoverying (reducing memory consumption)
+* osd: allow pg log trimming when degraded, scrubbing, recovering (reducing memory consumption)
```

Comment:
I don't think we need to correct these typos, it's just a title of the commit which have merged, and we can get the commit info by using `git log --grep=xxx`. eg.:
```
[root@ ]# git log --grep "ceph-volume: provide a nice errror message when missing ceph.conf"
commit 544bc3ea581230a824ca2742abd4ca346955a91d
Merge: f076960 0487a9a
Author: Alfredo Deza <alfredo@deza.pe>
Date:   Tue Jul 3 23:28:55 2018

    Merge pull request #22828 from ceph/wip-conf-error-message

    ceph-volume: provide a nice errror message when missing ceph.conf

    Reviewed-by: Alfredo Deza <adeza@redhat.com>
[root@ ]#
[root@ ]# git log --grep "ceph-volume: provide a nice error message when missing ceph.conf"
[root@ ]#

```